### DISCUSSION TITLE
Access to tags are missing?

### DISCUSSION THREAD

**Discussion by sorenpeter** [datenstrom/community#545] [Ask a question] [answered]
Hi,

It looks the the [blogtags] is not working anymore and I cannot find anything about showing tags in the current documentation at https://datenstrom.se/yellow/help/ 
How do I make links to tags or list them on my site?
I would like to access them via the url like this if possible: www.web.site/tags/TAGNAME

Best regards,
Søren Peter

**Comment by sorenpeter** [comment]
And i found the answer: http://www.darch.dk/tag:shaders
But it still seems to be undocumented;)

it is possible to make it using a slash instead of an colon, which will make it more familiar as a element in the URL?

/SP

**Comment by schulle4u** [comment, marked as answer]
The URL patterns are documented in the [blog extension documentation](https://github.com/datenstrom/yellow-extensions/tree/master/source/blog), see examples section for possible parameters. Maybe you can use rewrite roules to change the way Yellow handles URL parameters, AFAIK there's no way to directly configure it in Yellow. 

Steffen

**Comment by markseuffert** [comment]
The pattern looks like `/location/key:value/` and you'll see this in many extensions. There's a reason to use a colon character as a separator, it's probably obvious from the way I wrote it down. The slash character `/` separates files and folders, the colon character `:` separates keys and values. Datenstrom Yellow uses this convention in many places.

To answer your questions. No, it can't be configured, unless you switch to a completley different software. 😀

**Comment by sorenpeter** [comment]
Danke and tack both of your help both Mark and Steffen.
It works now. I just needed to add a / in [blogtags] - I presumed it had a default without a parameter.
And then I recalled correctly that I saw a place where this feature was explained, just missed that in all the gits.
I don't really need the domain/tag/myTag anyway with the above working. Just seen it somewhere else I think it is an easy  and neat way to access the content:)
